BookBinding.js

```diff
@@ -0,0 +1,42 @@
+import renderModule from './indexModule.js';
+
+class BookBinding {
+  constructor(localName) {
+    this.awesomeBooks = localStorage.getItem(`${localName}`) ? JSON.parse(localStorage.getItem(`${localName}`)) : [];
+    this.localName = localName;
+  }
+
+  add(title, author, bookshelf, classes) {
+    if (title.value.length && author.value.length) {
+      const book = new renderModule.Book(title.value.trim(), author.value.trim());
+      this.awesomeBooks.push(book);
```

Comment:
[OPTIONAL]
- [ ] Kindly considere using array methods that doesn't mutate the array, e.g. [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat), [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter), [slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) instead of [push](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push), [pop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/pop), [splice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice) etc.
In this case your code could look this : `this.awesomeBooks = this.awesomeBooks.concat(book)`